Articles/Setup/index.md

```diff
@@ -0,0 +1,25 @@
+---
+v8-equivalent: "https://github.com/umbraco/UmbracoDocs/blob/main/Getting-Started/Setup/index.md"
+versionFrom: 9.0.0
+verified-against: alpha-4
+state: partial
+updated-links: false
+---
+
+# Setup
+*How to install and configure your Umbraco installation*
+
+## [Requirements](Requirements/)
```

Comment:
No entirely sure about our structure, but shouldn't we link to the index.md page? So `Requirements/index.md`

Reply:
Once again, I have mentioned this already :) 
> Update the Umbraco.Templates nuget package link once we publish it in install-umbraco-with-templates.md;
It is currently referenced as https://www.nuget.org/packages/Umbraco.Templates;

Reply:
Nice thought, that's how we had it before, maybe Sofie knows best 🤷

Reply:
I think you're right @nikolajlauridsen !

At least in this repo we need to link directly to the index.md file.
On the original repo, we need only link to the folder, and it's set up to "automagically" choose the index file.

Reply:
Great! Will fix it